universal-login-relayer/lib/services/messages/PendingMessages.ts

```diff
@@ -45,7 +45,11 @@ export default class PendingMessages {
   }
 
   getMessageWithSignatures(message: SignedMessage, hash: string) : SignedMessage {
-    return  { ...message, signature: this.messagesStore.get(hash).getConcatenatedSignatures()};
+    const collectedSignatures = this.messagesStore.getCollectedSignatures(hash);
+    const sortedExecutions = sortExecutionsByKey([...collectedSignatures]);
```

Comment:
rename 
- `sortedExecutions` to `sortedSignaturesKeyPair`
- `sortExecutionsByKey` to `sortSignaturesKeyPair`